functionary/prompt_template/prompt_template_v2.py

```diff
@@ -10,6 +11,18 @@ class PromptTemplateV2(PromptTemplate):
     recipient_token = "<|recipient|>"
     content_token = "<|content|>"
     stop_token = "<|stop|>"
+    version = "v2"
+    # This token splits between function name and parameters
+    fn_param_sep_token = "\n<|content|> {"
+
+    def get_start_of_function_call_token(self) -> str:
+        return self.recipient_token
+
+    def get_stopping_token(self, stage: Literal["function", "parameter"]) -> int:
+        if stage == "function":
```

Comment:
Can we return string, then use tokenizer to get token_id, so we won't depend on the Model. In the future we might use another model, not Mistral

Reply:
The same for v1

Reply:
Gtocha